src/consensus/validator.rs

```diff
@@ -134,7 +134,8 @@ impl ShardValidator {
         ProposedValue {
             height: full_proposal.height(),
             round: full_proposal.round(),
-            validator_address: full_proposal.proposer_address(),
+            valid_round: full_proposal.round(),
```

Comment:
What's the different between round and valid_round?

Reply:
valid_round is the POL round. I think it should be set to `Nil` for all the cases in our code. It should be populated only for cases where we're restreaming an old proposal, not when we're proposing for the first time.